src/api/test-controller.js

```diff
@@ -107,10 +109,9 @@ export default class TestController {
             throw err;
         }
 
-        this.executionChain       = this.executionChain.then(() => this.testRun.executeCommand(command, callsite));
         this.callsiteWithoutAwait = callsite;
 
-        return this._createExtendedPromise(this.executionChain, callsite);
+        return this._createExtendedPromise(this.testRun.executeCommand(command, callsite), callsite);
```

Comment:
We manyally reject commands in `driverTaskQueue`.
It's the same like we had before for `executionChain`
https://github.com/DevExpress/testcafe/pull/621/files#diff-a21f56bee759db22062e32250e3df48cR235